src/OrchardCore.Modules/OrchardCore.Users/Migrations.cs

```diff
@@ -189,5 +190,22 @@ public int UpdateFrom9()
 
             return 10;
         }
+        
+        public int UpdateFrom10()
+        {
+            SchemaBuilder.AlterIndexTable<UserIndex>(table => table
+                .AddColumn<bool>(nameof(UserIndex.IsLockedOut), c => c.NotNull().WithDefault(false)));
+
+            SchemaBuilder.AlterIndexTable<UserIndex>(table => table
+                .AddColumn<DateTimeOffset?>(nameof(UserIndex.LockoutEnd), c => c.Nullable()));
+
+            SchemaBuilder.AlterIndexTable<UserIndex>(table => table
+                .AddColumn<int>(nameof(UserIndex.AccessFailedCount), c => c.NotNull().WithDefault(0)));
+
+            SchemaBuilder.AlterIndexTable<UserIndex>(table => table
+                .CreateIndex("IDX_UserIndex_IsLockedOut", "IsLockedOut"));
```

Comment:
This all needs to be in `Create` as well, and adjust create to skip this update.
The indexed column needs to be added to one of the existing indexes to be of some use ;)

Reply:
Are my latest changes in Migrations correct?

Reply:
Even you add these columns to an existing index table, better to also add them when defining this index table in `Create()`, so that on new install this index table is created in one pass. Then in `Create()` return 10 + 1 = 11.

To be okay with all providers, we should not use `DateTimeOffset` in the database, use `DateTime` instead

In the `OpenId` look at the `OpenIdAuthorizationIndexProvider` with a `DateTime` in the index / provider, and a `DateTimeOffset` in the `OpenIdAuthorizationStore` API, where we use the UTC part of the `DateTimeOffset`.

Reply:
@jtkech The version you are looking at might be outdated, I already did the changes you mention.

Reply:
@jtkech I can't use this syntax:
![image](https://user-images.githubusercontent.com/703248/117791387-91e59b80-b24a-11eb-8854-f9ab7a9fe4ba.png)
instead of:
`return Task.FromResult(new DateTimeOffset?(((User)user).LockoutEndUtc.Value));`
if it is what you mean.

Reply:
ok, @jtkech done.